docs/UserGuide.md

```diff
@@ -96,7 +103,7 @@ Section | Representation
 ---------|---------
 Menu Bar | The area where you can click to navigate and view some of TAB's features (e.g. help, reminder, and calendar). See [Menu Bar Shortcuts](#menu-bar-shortcuts) for more details.
 Menu Item | A button that you can click to view the stated feature, or to quit TAB (for <kbd>Exit</kbd>).
```

Comment:
I think you missed out this one

Reply:
This is a button on the UI so i didn't change